.github/workflows/merge.yml

```diff
@@ -56,6 +56,10 @@ jobs:
         name: coverage # optional
         fail_ci_if_error: true # optional (default = false)
         verbose: true # optional (default = false)
+
+    # Add -SNAPSHOT before deploy
+    - name: Add SNAPSHOT
+      run: mvn versions:set -DnewVersion='${project.version}-SNAPSHOT'
```

Comment:
We no longer maintain `-SNAPSHOT` in the POM because  Release Please will delete `-SNAPSHOT` every time it increments. So now instead we just add `-SNAPSHOT` in non-release builds dynamically.